hack/utils/tty.mli

```diff
@@ -38,8 +38,8 @@ type color_mode =
  * Print a sequence of colorized strings to stdout, using ANSI color escapes
```

Comment:
this comment is rotten given `stderr`.

you could alternatively pass an output channel and then call `Tty.cprint ~oc:stderr` or `Tty.cprint ~oc:stdout` instead of a boolean argument.